src/main/java/org/terasology/corerendering/rendering/dag/nodes/UpdateExposureNode.java

```diff
@@ -82,20 +80,16 @@ public void process() {
         if (renderingConfig.isEyeAdaptation()) {
             PerformanceMonitor.startActivity("rendering/" + getUri());
 
+            float[] pixels = new float[3];
+            writeOnlyPbo.readBackPixels(buffer -> {
+                pixels[0] = (buffer.get(2) & 0xFF) / 255.f;
+                pixels[1] = (buffer.get(1) & 0xFF) / 255.f;
+                pixels[2] = (buffer.get(0) & 0xFF) / 255.f;
```

Comment:
out of curiosity: why are those the wrong way round in the buffer?

Reply:
I haven't considered that wonder if its wrong.